### DISCUSSION TITLE
What is the capital of Burkina Faso?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#264] [Galaxy Trivia] [answered]
**Hint:** Ou-ou-ou

**Comment by tomkuttler** [comment]
The capital of Burkina Faso is Ouagadougou.

**Comment by ghost** [comment]
Uagadugú

**Comment by thuongtruong1009** [comment]
The capital of Burkina Faso is Ouagadougou.

**Comment by RexRowan** [comment]
Ouagadougou

**Comment by darko5r** [comment, marked as answer]
Ouagadougou, also spelled Wagadugu, capital and largest town of Burkina Faso, western Africa. It was the capital of the historic Mossi kingdom of Wagadugu (founded in the 15th century) and the seat of the morho naba (“great king”) of the Mossi people.

**Reply by ParanoidUser** [reply]
Bingo! One more to go for the Galaxy Brain badge. 😊

**Comment by Mithun1508** [comment]
Ouagadougou is the capital and administrative, communicative, cultural and economic center of Burkina Faso
